README.md

```diff
@@ -279,7 +279,7 @@ All wallet calls should be made with `$ hsw-rpc [call] [arguments...]`.
 
 - `getbids [name] [own]` - List own bids on a name.
 - `getreveals [name] [own]` - List own reveals on a name.
-- `getnames` - List all watched names and their statuses.
+- `getnames [own]` - List own names and their statuses.
```

Comment:
If https://github.com/handshake-org/hsd/pull/494 gets merged this may be a conflict. We can leave it for now and see how the other PR goes but either way the docs at https://github.com/handshake-org/handshake-org.github.io will need to be updated.

Reply:
I removed this and can update the docs if this gets merged